modules/update-upgrading-cli.adoc

```diff
@@ -59,7 +55,9 @@ For details and information on how to perform an `EUS-to-EUS` channel update, pl
 _Preparing to perform an EUS-to-EUS upgrade_ page, listed in the Additional resources section.
 ====
 
-. Based on your organization requirements, set the appropriate update channel. For example, you can set your channel to `stable-4.12`, `fast-4.12`, or `eus-4.12`. For more information about channels, refer to _Understanding update channels and releases_ listed in the Additional resources section.
+. Based on your organization requirements, set the appropriate update channel. For example, you can set your channel to `stable-4.13`. For more information about channels, refer to _Understanding update channels and releases_ listed in the Additional resources section.
```

Comment:
we can keep the reference to `fast-4.13` here.  It's everything from `Channel: ... (available channels: <anything from here>)`, with the caveat that [`candidate-4.y` channels will not limit you to supported releases][1].

[1]: https://docs.openshift.com/container-platform/4.13/updating/understanding-upgrade-channels-release.html#candidate-version-channel_understanding-upgrade-channels-releases